src/AppBundle/Twig/MarkdownExtension.php

```diff
@@ -24,7 +24,7 @@
  * @author Ryan Weaver <weaverryan@gmail.com>
  * @author Javier Eguiluz <javier.eguiluz@gmail.com>
  */
-class AppExtension extends \Twig_Extension
```

Comment:
there is a comment referring to this file in SourceCodeExtension. Please update it too